pkg/datapath/tables/node_address.go

```diff
@@ -383,6 +388,10 @@ func (n *nodeAddressController) updateFallbacks(txn statedb.ReadTxn, dev *Device
 		fallbacks.clear()
 		devices := n.Devices.All(txn)
 		for dev, _, ok := devices.Next(); ok; dev, _, ok = devices.Next() {
+			if strings.HasPrefix(dev.Name, "lxc") {
```

Comment:
question: would "lxc_" be a more specific prefix to look for?

Reply:
No:
```
root@kind-worker:/home/cilium# cilium statedb devices                                                                                                                                                                                                                                            # Name            Index   Selected   Type     MTU     HWAddr              Flags                    Addresses                                                                                                                                                                                     cilium_vxlan      4       false      vxlan    1500    46:1b:fc:10:41:ea   up|broadcast|multicast   fe80::441b:fcff:fe10:41ea                                                                                                                                                                     lxcd88bffc8e0a6   8       false      veth     1500    a2:3b:c8:f2:dc:16   up|broadcast|multicast   fe80::a03b:c8ff:fef2:dc16                                                                                                                                                                     lxc899f9560ba1b   10      false      veth     1500    ba:9e:f8:bc:ab:a7   up|broadcast|multicast   fe80::b89e:f8ff:febc:aba7                                                                                                                                                                     lxc509a2747c908   12      false      veth     1500    06:83:55:bd:c9:9b   up|broadcast|multicast   fe80::483:55ff:febd:c99b   
```